src/styles/report.css

```
@@ -30,3 +34,37 @@ h1 {
 	border-radius: 10px;
 	border: 2px solid black;
 }
+
+input, textarea {
+  width: 100%;
+  padding: 12px;
+  border: 1px solid #cccccc;
+  border-radius: 4px;
+  resize: vertical;
+}
+
+.container {
+  border-radius: 5px;
+  border: 5px solid #ED342F;
+  padding: 20px;
+  width: 75%;
+  margin: 0 auto;
+}
+
+label {
+  padding: 12px 12px 12px 0;
+  display: inline-block;
+}
+
+input {
+	width: 100%;
+	margin-top: 20px;
+}
+
+button[type="submit"]{
+	background: #ED342F;
+}
+
+button[type="submit"]:hover{
+	background: #A8569A;
+}
```

Let's namespace the selectors to `#report-form` (just a suggestion for the name), that way the styles don't "leak" everywhere else.

Also, moved the `input` and `textarea` styles to the namespace using the [`:is` selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:is) to make life easier.

```suggestion
#report-form {
  border-radius: 5px;
  border: 5px solid #ED342F;
  padding: 20px;
  width: 75%;
  margin: 0 auto;
}

#report-form :is(input, textarea) {
  width: 100%;
  padding: 12px;
  border: 1px solid #cccccc;
  border-radius: 4px;
  resize: vertical;
}

#report-form label {
  padding: 12px 12px 12px 0;
  display: inline-block;
}

#report-form input {
	margin-block-start: 20px;
}

#report-form button[type="submit"]{
	background: #ED342F;
}

#report-form button[type="submit"]:hover{
	background: #A8569A;
}
```

---

this is so awesome!!! definitely is way more specific! thank you!!